pkg/test/options.go

```diff
@@ -51,8 +52,9 @@ func Options(overrides ...OptionsFields) *azoptions.Options {
 		KubeletClientTLSBootstrapToken: lo.FromPtrOr(options.KubeletClientTLSBootstrapToken, "test-token"),
 		SSHPublicKey:                   lo.FromPtrOr(options.SSHPublicKey, "test-ssh-public-key"),
 		NetworkPlugin:                  lo.FromPtrOr(options.NetworkPlugin, "azure"),
-		NetworkPolicy:                  lo.FromPtrOr(options.NetworkPolicy, ""),
+		NetworkPolicy:                  lo.FromPtrOr(options.NetworkPolicy, "cilium"),
 		VMMemoryOverheadPercent:        lo.FromPtrOr(options.VMMemoryOverheadPercent, 0.075),
 		NodeIdentities:                 options.NodeIdentities,
+		SubnetID:                       DefaultVnetSubnetID,
```

Comment:
This makes it impossible to override, just follow the pattern above please

Reply:
fixed this